tests/Integration/bootstrap.php

```diff
@@ -52,3 +44,6 @@ class PHP_Token_NAME_FULLY_QUALIFIED extends PHP_Token {
 // Include the Parsely custom test cases.
 require_once __DIR__ . '/TestCase.php';
 require_once __DIR__ . '/StructuredData/NonPostTestCase.php';
+
+define( 'PARSELY_VERSION', '123456.78.9' );
```

Comment:
Does this break our checks to test the correct version?

Reply:
No, because that test get removed - it was redundant, as it wasn't really testing anything.

If anything, being able to define a random version, and still have the tests pass is a good thing, as it means we don't have logic hard-coded to expect specific version numbers.